src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -76,45 +71,67 @@ export const CodeExample: React.FC<{ code: string }> = ({ children, code }) => {
         css={css`
           display: flex;
           justify-content: flex-end;
-          background-color: ${p => getColor('grey', 200, p.theme)};
-          padding: ${p => p.theme.space.xxs} ${p => p.theme.space.xs};
+          border-top: ${p => p.theme.borders.sm} ${p => getColor('grey', 300, p.theme)};
+          background-color: ${p => getColor('grey', 100, p.theme)};
+          padding: ${p => p.theme.space.xxs} ${p => p.theme.space.sm};
         `}
       >
-        <Tooltip content={isRtl ? 'Enable LTR' : 'Enable RTL'}>
-          <StyledFooterButton onClick={() => setIsRtl(!isRtl)}>
-            {isRtl ? <DirectionRtlStroke /> : <DirectionLtrStroke />}
-          </StyledFooterButton>
+        <Tooltip content="Toggle RTL">
+          <ToggleIconButton
+            onClick={() => setIsRtl(!isRtl)}
+            isPressed={isRtl}
+            isPill={false}
+            focusInset
+          >
+            <DirectionRtlStroke />
+          </ToggleIconButton>
         </Tooltip>
         <form action="https://codesandbox.io/api/v1/sandboxes/define" method="POST" target="_blank">
           <input type="hidden" name="parameters" value={parameters} />
           <input type="hidden" name="query" value="module=src/Example.tsx" />
           <Tooltip content="Fork in Codesandbox">
-            <StyledFooterButton aria-label="Fork in Codesandbox" type="submit">
+            <IconButton
+              isPill={false}
+              focusInset
+              type="submit"
+              css={css`
+                margin-left: ${p => p.theme.space.sm};
+              `}
+            >
               <LightningBoltStroke />
-            </StyledFooterButton>
+            </IconButton>
           </Tooltip>
         </form>
         <Tooltip content={tooltipContent}>
-          <StyledFooterButton
-            aria-label="Copy to clipboard"
```

Comment:
Was this `aria-label` removal intended?

Reply:
Yes. The tooltip imparts `aria-describedby` to the button making `aria-label` redundant.